app/graphql/types/record_type.rb

```diff
@@ -205,6 +212,10 @@ def publication_date
       @object['dates']&.map { |date| date['value'] if date['kind'] == 'Publication date' }&.compact&.first
     end
 
+    def publication_information
+      @object['publishers']&.map { |publisher| publisher.values.join(', ') }
```

Comment:
Would you mind joining on `; `? I worry that a publisher name or location could have commas and be weird-ish.

`Publisher, Inc., December, 1, 1999, Boston, Ma` vs `Publisher, Inc.; December, 1, 1999; Boston, Ma`.

Not a blocker though as both suck and it's a deprecated field lol

Reply:
Good suggestion. I went ahead and made that change. (Didn't feel like it made sense to request another review, but feel free to confirm if you like!)